src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -310,8 +310,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </Target>
 
     <PropertyGroup>
-        <!-- Design-time builds require a newer version than 1.0 to succeed, so override back to inbox in that case -->
-        <CSharpCoreTargetsPath Condition="'$(CSharpCoreTargetsPath)' == '' or ('$(DesignTimeBuild)' == 'true' and $(CSharpCoreTargetsPath.Contains('Microsoft.Net.Compilers.1.0.0')))">$(RoslynTargetsPath)\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
+      <!-- If the SDK specified a path to a .NET Core version of the targets (from the SDK), use that so we align with their version but allow the user to override this. -->
+      <CSharpCoreTargetsPath Condition="'$(CSharpCoreTargetsPath)' == ''">$(CSharpCoreTargetsFromSdkPath)</CSharpCoreTargetsPath>
+
+      <!-- Design-time builds require a newer version than 1.0 to succeed, so override back to inbox in that case -->
+      <CSharpCoreTargetsPath Condition="'$(CSharpCoreTargetsPath)' == '' or ('$(DesignTimeBuild)' == 'true' and $(CSharpCoreTargetsPath.Contains('Microsoft.Net.Compilers.1.0.0')))">$(RoslynTargetsPath)\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
```

Comment:
`$(RoslynTargetsPath)` is also used in `UsingTask`s in `Microsoft.Common.tasks`. Is this override sufficient to get the task from the right place, or does more need to be done?

Reply:
Instead of changing `$(CSharpCoreTargetsPath)`, change `$(RoslynTargetsPath)`. This will change everywhere that points to the compiler resources, targets or not.